satpy/readers/abi_base.py

```diff
@@ -56,6 +57,10 @@ def __init__(self, filename, filename_info, filetype_info):
 
         self.coords = {}
 
+        if clip_negative_radiances is None:
+            clip_negative_radiances = satpy.config.get("clip_negative_radiances")
+        self.clip_negative_radiances = clip_negative_radiances
```

Comment:
Is there a reason we can overload `__init__` in the L1B file handler and only specify this kwarg there?

Reply:
Ok I added a `__init__` to the L1B file handler